packages/gatsby/migrations.json

```diff
@@ -145,6 +145,87 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "12.3.7": {
+      "version": "12.7.0-beta.0",
```

Comment:
You probably meant `12.3.7-beta.0` :sweat_smile:

Reply:
yes.

Reply:
changed.